### ISSUE TITLE
Cannot Begin Ongoing Tasks

### ISSUE THREAD

**Issue by RaymondO55** [over55/workery-django#489]
There is not a way to begin the ongoing tasks due to the lack of a begin button. Image provided for clarification.
![ongoing task](https://user-images.githubusercontent.com/41381131/43780655-12c0fa34-9a29-11e8-909e-b84ec9f1dfb1.png)

**Comment by RaymondO55** [comment]
**Update**

This seems to only be an issue with old job orders before the process was completely implemented. I will release a list in regards to the status of these job orders.

**Comment by bartmika** [comment]
@RaymondO55 Thank you for the update and opening up this bug issue. I confirm this was created by issue #428. I will investigate the problem.

**Comment by bartmika** [comment]
@RaymondO55 May I please get more details pertaining to how this error occurred? My suspision is that these where created in the previous code (which we removed via #428). A quick solution for us is to delete all of these tasks since they belong to the old codebase. Would this be an acceptable issue for your team? Thanks!

**Comment by RaymondO55** [comment]
@bartmika Yes please proceed with the removal of these tasks. We will recreate new master ongoing job orders as we see fit with the new code to account for the ongoing job invoices. Thanks

**Comment by bartmika** [comment]
@RaymondO55 Hi Raymond, task has been completed. Please confirm and close ticket when ready.
